infrastructure/ansible/roles/dataset_loader/defaults/main.yml

```diff
@@ -21,6 +21,8 @@ dl_ts_url: https://stats.kabletown.invalid
 # Credentials to use when logging into TrafficOps
 dl_to_user:
 dl_to_user_password:
+# Federation User account name
+federation_user: federationuser
```

Comment:
Unprefixed vars in roles aren't best practice.  Just set the value in `dl_ds_default_federation_user`.

Reply:
Setting the value in `dl_ds_default_federation_user` in 0cf68cdc3d.